_commands/basics/seq.md

```diff
@@ -3,22 +3,64 @@
 
 seq
 -------
-TODO: Add documentation for this command by submitting a pull request.
+`seq` is used to produce a sequence of numbers
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ seq 3
+1
+2
+3
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `seq [FIRST] [LAST]`
+
+~~~ bash
+$ seq 5 10
+5
+6
+7
+8
+9
+10
+~~~
+
+##### Break it down
+ * Putting in two number arguments prints a list of numbers starting from the 
+first number to the second number
+
+#### `seq [FIRST] [INCREMENT] [LAST]`
+
+~~~ bash
+$ seq 2 2 10
+2
+4
+6
+8
+10
+~~~
 
 ##### Break it down
+ * The `FIRST` argument is where the sequence will start. 
+ * The `INCREMENT` argument determines how the sequence increments by.
+ * The `LAST` argument is where the sequence will end.
 
-#### Example command
+#### Bash Script
```

Comment:
If possible, find a way to make the distinction between the script and the output produced by the script more clear. You could do this by separating them into different code blocks or by including comments somewhere.